src/pages/Newsfeed.js

```diff
@@ -1,14 +1,16 @@
 import Postcard from "../components/Postcard/Postcard";
 import Stories from "../components/Stories/Stories";
+import { Button } from "@mui/material";
 import { useState } from "react";
 import { useEffect } from "react";
+import { loadMorePosts } from "../services/GetPosts";
 
 const Newsfeed = () => {
   const [posts, setPosts] = useState([]);
   useEffect(() => {
     async function getPosts() {
       const response = await fetch(
-        process.env.REACT_APP_EXPRESS_URL + `/posts/`
+        process.env.REACT_APP_EXPRESS_URL + `/posts/paged/?lastPost=&size=`
```

Comment:
confused here. are you just sending those as empty on this call ? why bother including them